stories/CodeBlock.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+import { action } from '@storybook/addon-actions';
+import { StoryContainer } from './helpers';
+
+import { CodeBlock, Panel } from '../src';
+
+storiesOf('Code Block', module)
+  .addDecorator((getStory) => (
+    <StoryContainer>{ getStory() }</StoryContainer>
+  ))
+  .add('without chevrons', withInfo()(() => (
+    <Panel sectioned>
+    <CodeBlock code={`curl -X POST
+https://api.sparkpost.com/api/v1/transmissions
+-H "Authorization: cf85ca1c884bd13624fadea62ec21cc5cd477126"
```

Comment:
i hope that's not a real key, i'd just put <API_KEY> or something

Reply:
ah crap. Deleted the associated api key